Puzzlepart/prosjektportalen365
# Added Porteføljeinnsyn group and button in configuration page

### Your checklist for this pull request

- [x] Remember: After PR is closed, add PR description to [Changelog](https://github.com/Puzzlepart/prosjektportalen365/blob/dev/CHANGELOG.md)

### Description

Renames PortfolioManagerGroupName string from Porteføljeinnsikt to Porteføljeinnsyn
Adds a Porteføljeinnsyn group utilized by #243 
Adds configuration button
![image](https://user-images.githubusercontent.com/10527497/93568876-17050300-f991-11ea-850a-3639ed65c219.png)

### How to test

- Go to {site}/SitePages/Konfigurasjon.aspx
- Add a user to the group
- Check Porteføljeoversikt if projects are visible

### Relevant issues (if applicable)
#243 

💔Thank you!
